pkg/assetfetcher/stater_mock_test.go

```diff
@@ -0,0 +1,132 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/asecurityteam/nexpose-asset-producer/pkg/domain (interfaces: Stat)
+
+// Package assetfetcher is a generated GoMock package.
+package assetfetcher
+
+import (
+	gomock "github.com/golang/mock/gomock"
+	reflect "reflect"
+	time "time"
+)
+
+// MockStat is a mock of Stat interface
+type MockStat struct {
```

Comment:
suggestion: This package now has files called `mock_stater_test.go` and `stater_mock_test.go`. The `mock_<interface>_test.go` is a convention we use regularly in our code and it's odd to see a one-off that changes the pattern.

Personally, I think called the _other_ file `mock_stater_test.go` is less accurate than calling _this_ file `mock_stater_test.go` since this one actually contains a mock. The other file contains something closer to what folks would consider a stub. I suggest renaming the other file to `stub_stat_test.go` and this one to `mock_stat_test.go` to both clarify the intentions and make the naming scheme more standard.

Reply:
that's a good point, i like that idea.

Reply:
changed!